tools/acrn-crashlog/README.rst

```diff
@@ -0,0 +1,124 @@
+ACRN-Crashlog
+#############
+
+Introduction
+************
+
+The ``ACRN-Crashlog`` is a debug tool to collect any useful logs/information after each crash/event occurred on the platform, including hypervisor, Service OS(SOS) and Android as a Guest(AaaG). It provides abundant logs for developers to debug/analyze the crashes. And it provides a flexible way to configure which crash/event to be concerned with xml file.
+
+Building
+********
+
+Build dependencies
+==================
+
+The ``ACRN-Crashlog`` tool depends on the following libraries (build and runtime):
+
+- libevent
+
+- OpenSSL
+
+- libxml2
+
+- systemd
+
+- telemetrics-client-dev (optional, detected at build time)
+
+For more detailed instructions on how to set-up your build environment, please refer to the :ref:`getting_started`.
+
+Build
+=====
+
+To build the ``ACRN-Crashlog``
+
+.. code-block:: console
+
+   make
+
+To Clean the build:
+
+.. code-block:: console
+
+   make clean
+
+Installing
+**********
+
+Install dependencies
+====================
+
+Since ``ACRN-Crashlog`` runs as a probe of `telemetrics-client`_, it needs `telemetrics-client`_ (>=1.6.0) to be available on the system.
+
+Install
+=======
+
+To install the build
+
+.. code-block:: console
+
+   sudo make install
+
+Usage
+*****
+
+``ACRN-Crashlog`` works as a probe of `telemetrics-client`_. It is autostarted, running as daemon and sending the crashlog path to `telemetrics-client`_.
+`telemetrics-client`_ records events of interest and reports them to the backend using telemd telemetrics daemon. The type of events/logs is implemented as probe. The work flow of acrnprobe and `telemetrics-client`_ is:
+
+.. code-block:: console
+
+            crashlog path                   log content
+   acrnprobe------------->telemetrics-client----------->backend
+
+For more details of telemetrics, please refer the `telemetrics-client`_ and `telemetrics-backend`_ website.
+
+Crashlog can be retrieved with ``telem_journal`` command:
+
+.. code-block:: console
+
+   telem_journal -i
+
+``ACRN-Crashlog`` also provides a tool debugger to dump the specific process information:
```

Comment:
``debugger`` seems like the wrong name to use for this tool if all it does is dump process information.

Reply:
We inherit the name from an android name which named "debuggerd". Could we rename it in the further patches? Since we need to modify the code, file name, makefile and etc.